src/ui/MLFacet/MLFacet.ts

```diff
@@ -109,14 +109,14 @@ export class MLFacet extends Component {
     }),
 
     /**
-     * The maximum number of values to request for this facet.
+     * The number of values to request for this facet.
      *
-     * **Minimum:** `0`
+     * Also determines the maximum number of additional values to request each time this facet is expanded,
```

Comment:
Also determines the maximum number -> Also determines the default maximum number